extensions/amp-instagram/0.1/amp-instagram.js

```diff
@@ -170,9 +140,7 @@ class AmpInstagram extends AMP.BaseElement {
       removeElement(this.iframe_);
       this.iframe_ = null;
       this.iframePromise_ = null;
-      setStyles(this.placeholderWrapper_, {
-        'display': '',
-      });
+      this.togglePlaceholder(true);
```

Comment:
This has to go into the runtime level. Whoever calls `unlayoutCallback` should also toggle placeholder back on.